site/src/components/samplers/modeMedian.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: component
+title: Mode Median Sampling
+component: data/samplers/modeMedian.js
+namespace: samplers
+
+example-code: |
+    var data = fc.data.random.financial()(10000);
+
+    var subsampledData = fc.data.samplers.modeMedian()
+                                         .number(5)
+                                         .field('low')(data);
+
+    d3.select('#subsampled-data')
```

Comment:
Can you also show the original data on this page? I think it will provide a much better illustration of the functionality. You might need to reduce the points from 10,000 to something more manageable!